docs/src/main/sphinx/connector/hive.rst

```diff
@@ -390,51 +398,97 @@ In order to use a Hive Thrisft metastore, you must configure the metastore with
 properties:
 
 .. list-table:: Thrift metastore configuration properties
-   :widths: 50 50
+   :widths: 35, 50, 15
    :header-rows: 1
 
-   * - Property Name
+   * - Property name
      - Description
+     - Default
```

Comment:
A convention we chose for a lot of table is to NOT have a default column because a lot of properties dont have a default defined..

Reply:
I think it makes sense, because this is adding a lot of rows to the table that do have default values.